docs/user-guide/options.md

```diff
@@ -40,12 +40,12 @@ For CSS with standard syntax, Stylelint uses [postcss-safe-parser](https://www.n
 
 When using the Node.js API, the autofixed code is available as the value of the `code` property in the returned object.
 
-If a source contains a:
+---
```

Comment:
[suggest] Minor. Personally, I feel this bar is a bit verbose. Each heading has an underline, so I am concerned that people may be a bit confused about which section is separated.

```suggestion
```